src/pages/Home.jsx

```diff
@@ -39,72 +46,24 @@ function Home({ location }) {
 
             <div className="mt-10">
               <dl className="space-y-10 md:space-y-0 md:grid md:grid-cols-2 md:gap-x-8 md:gap-y-10">
-                <div className="flex">
-                  <div className="flex-shrink-0">
-                    <div className="flex items-center justify-center h-12 w-12 rounded-md bg-indigo-500 text-white">
-                      <FontAwesomeIcon icon={faGlobe} className="text-white" />
+                {layout.map(box => (
```

Comment:
whenever there is a repeated structure, it's nice to create a map(dada=>, because then, if something needs to be fixed, it is only done in one place

Reply:
Indeed :)